contributed_definitions/nyaml/NXmpes.yaml

```
@@ -323,39 +524,174 @@ NXmpes(NXobject):
+              Axes should be named according to the conventions defined below. Note that this
+              list is a glossary with explicitly named axis names, which is only intended to
+              cover the most common measurement axes and is therefore not complete. It is
+              possible to add axes with other names at any time.
             \@signal:
               enumeration: [raw]
             raw(NX_NUMBER):
               doc: |
                 Raw data before calibration.
+            pixel_x(NX_NUMBER):
+              exists: optional
+              unit: NX_ANY
+              doc: |
+                Detector pixel in x direction.
+            pixel_y(NX_NUMBER):
+              exists: optional
+              unit: NX_ANY
+              doc: |
+                Detector pixel in y direction.
+            energy(NX_NUMBER):
+              exists: recommended
+              unit: NX_ENERGY
+              doc: |
+                (Un)calibrated energy axis.
+              \@type(NX_CHAR):
+                doc: |
+                  The energy can be either stored as kinetic or as binding energy.
+                enumeration:
+                  kinetic:
+                    doc:
+                    - |
+                      (Un)calibrated kinetic energy axis.
+                    - |
+                      xref:
+                        spec: ISO 18115-1:2023
+                        term: 3.35
+                        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+                  binding:
+                    doc:
+                    - |
+                      (Un)calibrated binding energy axis.
+                    - |
+                      xref:
+                        spec: ISO 18115-1:2023
+                        term: 12.16
+                        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+            photon_energy(NX_NUMBER):
+              exists: optional
+              unit: NX_ENERGY
+              doc: |
+                (Un)calibrated photon energy of the incoming probe beam.
+                
+                Could be a link to /entry/instrument/beam_probe/incident_energy.
+            kx(NX_NUMBER):
+              exists: optional
+              unit: NX_WAVENUMBER
+              doc: |
+                (Un)calibrated k-space coordinate in x direction. It is envisioned that the axes in
+                momentum space are named ``kx``, ``ky``, and ``kz``. Typically, the vectors in
+                momentum space are defined such that ``kx`` and ``ky`` comprise the parallel
+                component, while ``kz`` is the perpendicular component.
+                
+                It is also possible to define ``k_parallel`` and ``k_perpendicular`` for the parallel
+                and perpendicular momenta, respectively.
+                
+                Units are typically 1/Angström.
```

I think it should be `angstrom`, lower case and without umlaut. At least pint complains about "Angström", and also "Angstrom"
```suggestion
                Units are typically 1/angstrom.
```

---

Okay, makes sense. Implemented.